akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1466,13 +1472,18 @@ class MarkerLoggingAdapter(
   def this(bus: LoggingBus, logSource: String, logClass: Class[_]) =
     this(bus, logSource, logClass, new DefaultLoggingFilter(() ⇒ bus.logLevel))
 
+  def isErrorEnabled(marker: LogMarker) = loggingFilter.isErrorEnabled(logClass, logSource, marker)
+  def isWarningEnabled(marker: LogMarker) = loggingFilter.isWarningEnabled(logClass, logSource, marker)
+  def isInfoEnabled(marker: LogMarker) = loggingFilter.isInfoEnabled(logClass, logSource, marker)
+  def isDebugEnabled(marker: LogMarker) = loggingFilter.isDebugEnabled(logClass, logSource, marker)
+
```

Comment:
add similar in https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/scala/akka/actor/typed/Logger.scala#L170
and https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/LoggerAdapterImpl.scala#L366

Reply:
done